meson.build

```diff
@@ -19,11 +19,11 @@ sources = [
 dependencies = {
     'granite':    dependency('granite'),
     'indicator3': dependency('indicator3-0.4'),
-    'wingpanel2': dependency('wingpanel-2.0')
+    'wingpanel3': dependency('wingpanel')
```

Comment:
I'm not sure if I right but i think package name should be `io.elementary.wingpanel` because of recent renaming in wingpanel repository